hgloves/src/components/computer/email/email.js

```diff
@@ -0,0 +1,178 @@
+//Imports
```

Comment:
I guess we already know lines above are import, not sure this comment is necessary.